Dockerfile

```diff
@@ -64,8 +64,8 @@ COPY ironic-config/apache2-ipxe.conf.j2 /etc/httpd-ipxe.conf.j2
 
 # DATABASE
 RUN mkdir -p /var/lib/ironic && \
-  sqlite3 /var/lib/ironic/ironic.db "pragma journal_mode=wal" && \
-  dnf remove -y sqlite
+    sqlite3 /var/lib/ironic/ironic.sqlite "pragma journal_mode=wal" && \
```

Comment:
All indentation in this file is 2 spaces, please don't change that.

Reply:
Thanks for the review.
It has been updated
regards,